### PR TITLE
[it] Localizzazione termine Virtualization

### LINKED ISSUE #1658: [it] Localizzazione Virtualization
https://glossary.cncf.io/virtualization/

### PR DESCRIPTION
**Pull request by ugho16**
### Describe your changes
Added Virtualization italian localization

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1658 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/it/virtualization.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+category: tecnologia
+tags: ["fondamentali", "infrastruttura", "metodologia"]
+---
+
+## Cos'è
+
+Nel contesto del cloud native computing, la virtualizzazione
+si riferisce al processo di prendere un computer fisico, a volte chiamato server,
+e permettergli di eseguire diversi sistemi operativi isolati tra loro.
+Questi sistemi operativi e le loro risorse computazionali dedicate (CPU, memoria e rete) sono
```
Comment (fsbaraglia) [inline content/it/virtualization.md:13]:
```suggestion
Tali sistemi operativi isolati e le risorse di calcolo dedicate ad essi (CPU, memoria e rete) vengono
```

File: content/it/virtualization.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+chiamati macchine virtuali (VM).
```
Comment (fsbaraglia) [inline content/it/virtualization.md:14]:
```suggestion
chiamati macchine virtuali o VM.
```

File: content/it/virtualization.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Virtualizzazione
+status: completed
+category: tecnologia
+tags: ["fondamentali", "infrastruttura", "metodologia"]
+---
+
+## Cos'è
+
+Nel contesto del cloud native computing, la virtualizzazione
```
Comment (fsbaraglia) [inline content/it/virtualization.md:10]:
```suggestion
La virtualizzazione, nell'ambito del calcolo nativo su cloud, 
```

File: content/it/virtualization.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+si riferisce al processo di prendere un computer fisico, a volte chiamato server,
```
Comment (fsbaraglia) [inline content/it/virtualization.md:11]:
```suggestion
fa riferimento al processo di prendere un computer fisico, talvolta denominato server, 
```

File: content/it/virtualization.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+e permettergli di eseguire diversi sistemi operativi isolati tra loro.
```
Comment (fsbaraglia) [inline content/it/virtualization.md:12]:
```suggestion
e consentirgli di eseguire più sistemi operativi isolati.
```

File: content/it/virtualization.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## Quali problematiche affronta
+
+La virtualizzazione affronta molte problematiche, tra cui il miglioramento dell'utilizzo dell'hardware fisico,
+in quanto permette a più applicazioni di essere eseguite sulla stessa macchina fisica
+mantenendole isolate l'una dall'altra per sicurezza.
+
+## In che modo aiuta
+
+Le applicazioni eseguite su macchine virtuali non sono consapevoli di star condividendo un computer fisico.
```
Comment (fsbaraglia) [inline content/it/virtualization.md:28]:
```suggestion
Le applicazioni eseguite su macchine virtuali operano senza alcuna consapevolezza del fatto che condividono una stessa risorsa fisica.
```

File: content/it/virtualization.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+La virtualizzazione permette anche agli utenti del [datacenter](/it/datacenter/) di avviare un nuovo "computer" (cioè una VM) in pochi minuti,
+senza preoccuparsi dei vincoli fisici di dover aggiungere un nuovo computer fisico al datacenter.
+Le VM, inoltre, permettono agli utenti di ridurre il tempo necessario per avere a disposizione un nuovo computer virtuale.
```
Comment (fsbaraglia) [inline content/it/virtualization.md:31]:
```suggestion
Le VM consentono inoltre agli utenti di ridurre il tempo necessario per mettere a disposizione un nuovo computer.
```

**Review by fsbaraglia — CHANGES_REQUESTED** [review]

**Review by fsbaraglia — APPROVED** [review]

**Comment by fsbaraglia** [comment]
@annalisag-spark  please review

File: content/it/virtualization.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
+---
+title: Virtualizzazione
+status: completed
+category: tecnologia
+tags: ["fondamentali", "infrastruttura", "metodologia"]
+---
+
+## Cos'è
+
+La virtualizzazione, nell'ambito del calcolo nativo su cloud,
```
Comment (annalisag-spark) [inline content/it/virtualization.md:10]: "calcolo nativo sul cloud" mi suona molto curioso. sento sempre utilizzare la versione in inglese. manterrei quella con la traduzione in italiano tra parentesi, al massimo.

File: content/it/virtualization.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+title: Virtualizzazione
+status: completed
+category: tecnologia
+tags: ["fondamentali", "infrastruttura", "metodologia"]
+---
+
+## Cos'è
+
+La virtualizzazione, nell'ambito del calcolo nativo su cloud, 
+fa riferimento al processo di prendere un computer fisico, talvolta denominato server,
```
Comment (annalisag-spark) [inline content/it/virtualization.md:11]:
```suggestion
si riferisce alla pratica per la quale a un computer fisico, talvolta denominato server,
```

File: content/it/virtualization.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+status: completed
+category: tecnologia
+tags: ["fondamentali", "infrastruttura", "metodologia"]
+---
+
+## Cos'è
+
+La virtualizzazione, nell'ambito del calcolo nativo su cloud, 
+fa riferimento al processo di prendere un computer fisico, talvolta denominato server, 
+e consentirgli di eseguire più sistemi operativi isolati.
```
Comment (annalisag-spark) [inline content/it/virtualization.md:12]:
```suggestion
viene consentito di eseguire più sistemi operativi isolati.
```

File: content/it/virtualization.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+Tali sistemi operativi isolati e le risorse di calcolo dedicate ad essi (CPU, memoria e rete) vengono
```
Comment (annalisag-spark) [inline content/it/virtualization.md:13]:
```suggestion
Tali sistemi operativi isolati e le risorse di calcolo ad essi dedicate (CPU, memoria e rete) vengono
```

File: content/it/virtualization.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+chiamati macchine virtuali o VM.
+Quando si parla di [macchine virtuali](/it/virtual-machine/), si fa riferimento a un computer definito da un software.
+Qualcosa che sembra e agisce come un vero computer ma che condivide l'hardware con altre macchine virtuali.
+Il [cloud computing](/it/cloud-computing/) è alimentato principalmente dalle tecnologie di virtualizzazione.
+Per esempio, è possibile noleggiare un "computer" da AWS, ma quel computer è in realtà una VM.
```
Comment (annalisag-spark) [inline content/it/virtualization.md:18]: qui preferirei rimanere vendor agnostic e sostituire "da AWS" con "dal cloud provider"

File: content/it/virtualization.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+
+La virtualizzazione affronta molte problematiche, tra cui il miglioramento dell'utilizzo dell'hardware fisico,
+in quanto permette a più applicazioni di essere eseguite sulla stessa macchina fisica
+mantenendole isolate l'una dall'altra per sicurezza.
+
+## In che modo aiuta
+
+Le applicazioni eseguite su macchine virtuali operano senza alcuna consapevolezza del fatto che condividono una stessa risorsa fisica.
+La virtualizzazione permette anche agli utenti del [datacenter](/it/datacenter/) di avviare un nuovo "computer" (cioè una VM) in pochi minuti,
+senza preoccuparsi dei vincoli fisici di dover aggiungere un nuovo computer fisico al datacenter.
```
Comment (annalisag-spark) [inline content/it/virtualization.md:30]:
```suggestion
senza preoccuparsi dei vincoli fisici di dover aggiungere del nuovo hardware al datacenter.
```

File: content/it/virtualization.md (hunk 12 above)
Comment (ugho16) [inline content/it/virtualization.md:18]:
```suggestion
Per esempio, è possibile noleggiare un "computer" da un provider, ma quel computer è in realtà una VM.
```

File: content/it/virtualization.md (hunk 8 above)
Comment (ugho16) [inline content/it/virtualization.md:10]:
```suggestion
La virtualizzazione, nell'ambito del cloud native computing (calcolo nativo su cloud), 
```

**Review by annalisag-spark — APPROVED** [review]

**Pull request merged**